core/arch/arm/kernel/secure_partition.c

```diff
@@ -581,6 +586,7 @@ static TEE_Result load_binary_sp(struct ts_session *s,
 	return res;
 }
 
+__pauth
 static TEE_Result sp_open_session(struct sp_session **sess,
```

Comment:
Attributes should come after the return type
```suggestion
static TEE_Result __pauth
sp_open_session(struct sp_session **sess,
```

Reply:
Fixed